docs/development/new-packages.md

```diff
@@ -133,6 +133,30 @@ find ways to include your patches into the package. Many package maintainers are
 very receptive to including Pyodide-related patches and they reduce future
 maintenance work for us.
 
+### 4. Try out locally built package
+
+On a successful local build you should now have a wheel
+`<package-name>-<version>-cp310-cp310-emscripten_wasm32.whl` in directory
+`packages/<package-name>/dist/`.
+
+You can download the
+[current pyodide build](https://github.com/pyodide/pyodide/releases), unpack it
+in a location of your choice and mix in your locally built wheel. Copy your
+wheel into the unpacked directory and manually add the appropriate information
+to the `packages.json` file in a text editor of your choice (e.g.
+`"<package-name>": {"name": "<package-name>", "version": "<version>", "file_name": "<package-name>-<version>-cp310-cp310-emscripten_wasm32.whl", "install_dir": "site", "depends": ["setuptools", "numpy", "scipy"], "imports": ["<package-name>"]}`).
```

Comment:
I'm not sure I would recommend this workflow. It's not wrong or anything, it just seems like a lot of trouble. Ideally we should make it easier for people to generate the right setup with `pyodide_build`.

Reply:
I totally agree, ideally it should be easier. This is just my view, what I would've liked to see on that page, to help make sense after getting the wheel built. Coming in on the pyscript hype train with zero knowledge of pyodide (but suffering through Debian python packaging and doing conda-forge feedstocking loads).

I was looking at this wheel I managed to build after a while and took me a few hours to figure out how to use it.

Reply:
Right now, the guideline kinda ends abruptly when the wheel is supposed to be built. And leaves you wondering how to use it.

Reply:
Absolutely feel free to ignore this PR in favor of a better solution just thought I might as well take 5 mins to share my view on the docs after a few hours of making sense of things =)

Reply:
> as some weird parts but nothing as bad as manually writing out the `packages.json`

:rofl: yeah, well. comin in to something absolutely new, can't hurt to be a vi tinkerer.

Like I said, just close this, but if you do so, put something else in fast. I bet you get a lot of people looking at that page right now, that know nothing of pyodide, but been packaging python modules for years.

Reply:
Anything you put in about what to do with that locally built wheel is better than the nothing whats in there right now :)